client/src/app/pages/applications/components/application-identity-form/application-identity-form.tsx

```diff
@@ -105,7 +99,7 @@ export const ApplicationIdentityForm: React.FC<
           id: application.id,
           businessService: application.businessService,
         };
-        let promise: AxiosPromise<Application>;
+        let promise: Promise<Application>;
```

Comment:
What is the difference here?

Reply:
Because `updateApplication` returns a Promise. 
Inside `axios.put()` returns a Promise<Application>`